tests/utils/replication_slots.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright The CloudNativePG Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/cloudnative-pg/cloudnative-pg/pkg/specs"
+)
+
+// CompareLsn returns true if all the LSN values inside a given list are the same
+func CompareLsn(lsnList []string) bool {
+	for _, lsn := range lsnList {
+		if lsn != lsnList[0] {
+			return false
+		}
+	}
+	return true
+}
+
+// DiscoverRepSlotsOnPod returns a list of replication slot names which should be present
+// in a given pod
+func DiscoverRepSlotsOnPod(namespace, clusterName, podName string, env *TestingEnvironment) ([]string, error) {
```

Comment:
This function returns the expected replication slot names in a pod through name converting, should we use getExpectedRepSlotOnPod.

Reply:
Done.